src/Robo/Commands/Setup/ConfigCommand.php

```diff
@@ -26,6 +26,8 @@ public function update() {
    * @command drupal:config:import
    * @aliases dci setup:config-import
    *
+   * @executeInDrupalVm
+   *
```

Comment:
Should we also add this to `drupal:update`?

Reply:
@grasmash i'm not sure, adding the above fixed sync for me. i'm not sure if it is needed elsewhere or not?